lib/manager/nuget/readme.md

```diff
@@ -2,4 +2,4 @@ The `nuget` configuration object is used to control settings for the NuGet packa
 The NuGet package manager supports a SDK-style `.csproj`/`.fsproj`/`.vbproj`/`.props`/`.targets` format, as described [here](https://natemcmaster.com/blog/2017/03/09/vs2015-to-vs2017-upgrade/).
 This means that .NET Core projects are all supported but any .NET Framework projects need to be updated to the new `.csproj`/`.fsproj`/`.vbproj`/`.props`/`.targets` format in order to be detected and supported by Renovate.
 
-The NuGet manager also supports `global.json` and `.config/dotnet-tools.json` SDK files.
+The NuGet manager also supports `global.json` and `.config/dotnet-tools.json`/`dotnet-tools.json` SDK files.
```

Comment:
Maybe we don't have to mention `.config` at all here, we want to support `dotnet-tools.json` in all valid locations.

```suggestion
The NuGet manager also supports `global.json` and `dotnet-tools.json` SDK files.
```